testing/directives-test.js

```diff
@@ -1,15 +1,14 @@
-/*
 (function(){
    'use strict';
-    describe('Game square tests', function(){
+    describe('Directive tests', function(){
        var compile;
        var rootScope;
-
-        beforeEach(module('tombola.noughtsAndCrossesApp.directive'));
+       var gameSquare;
+       var squareNumber;
 
         beforeEach(
             module(function($provide){
-                $provide.value('gameSquare', function(value){
+                $provide.value('tombola.noughtsAndCrossesApp.directive' ,'gameSquare', function(value){
```

Comment:
This line doesn't mak sense: You probably want to provide 'gameSquare' and then an instance of a mock.
What it's currently doing is providing the string 'gameSquare' when 'tombola.noughtsAndCrossesApp.directive' is requested from the DI. The third parameter (the function) is not used by the provider.